src/components/SelectionBox/SelectionBox.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+
+const COLOR_SET = Object.freeze({
+  normalPink: '#FF8585',
+  normalGreen: '#3BAB29',
+  normalBlack: '#2F3438',
+  disabled: '#aaaaaa',
+  brand: '#2A2E56'
+});
+
+const SelectionBox = ({ options, defaultOption, required, disabled, fontSize, height }) => {
+  const [color, setColor] = useState(decideColor({ required, disabled }));
+
+  const changeHandler = (e) => {
+    const chosen = e.target.selectedIndex !== 0;
+    const nextColor = decideColor({ required, chosen, disabled });
+    setColor(nextColor);
+  };
+
+  return (
+    <Wrapper>
+      <SelectionWrapper color={color}>
+        <Selection
+          onChange={changeHandler}
+          disabled={disabled}
+          color={color}
+          fontSize={fontSize}
+          height={height}>
+          <Option>{defaultOption}</Option>
+          {options.map((content, index) => (
+            <Option key={index}>{content}</Option>
```

Comment:
배열의 인덱스를 key 값으로 사용하는 것은 안티 패턴으로 알고 있습니다. 서버에서 받은 데이터 안에 있는 id 값을 key로 사용해야 할 것 같아요. https://ko.reactjs.org/docs/reconciliation.html#keys

```suggestion
          {options.map({id, option}) => (
            <Option key={id}>{option}</Option>
```